backend/app/views/spree/admin/cms_sections/types/_image_gallery.html.erb

```diff
@@ -50,16 +50,17 @@
       <% end %>
 
       <div class="col-12"><hr></div>
-
-      <%= f.field_container :link_type_one, class: ['form-group'] do %>
-        <%= f.label :link_type_one, Spree.t('admin.navigation.link_to') %>
-        <%= f.select(:link_type_one,
-            spree_humanize_dropdown_values('Spree::Cms::Sections::ImageGallery',
-                                            { const: 'LINKED_RESOURCE_TYPE' }),
-                                            { include_blank: false },
-                                            class: 'link_switcher',
-                                            data: { target_field: :link_one}) %>
-        <%= f.error_message_on :link_type_one %>
+      <% if Rails::VERSION::STRING > '5.9' %>
```

Comment:
@MatthewKennedy let's create a helper with the method `rails_5?`

Reply:
Sure not a problem.